github-dark.css

```diff
@@ -2512,6 +2512,61 @@
   html.octotree:not(.octotree-show) .zh-board-expanded .zh-board-menu .tabnav-left {
     margin-left: 20px;
   }
+
+  /* Awesome autocomplete styling */
+  .awesome-autocomplete .tt-dropdown-menu {
+    border-color: #343434 !important;
+    background-color: #181818 !important;
+  }
+  .awesome-autocomplete .aa-query strong,
+  .awesome-autocomplete .aa-query-default em {
+    color: /*[[base-color]]*/ #4183c4 !important;
+  }
+  .awesome-autocomplete .aa-category {
+    background-color: #1d1d1d !important;
+    border-top: 1px solid #343434 !important;
+    border-bottom: 1px solid #343434 !important;
+    color: #a0a0Aa !important;
+  }
+  .awesome-autocomplete .tt-suggestion {
+    border-color: #343434 !important;
+  }
+  .awesome-autocomplete .tt-suggestion .octicon {
```

Comment:
You mean with the global definitions, those huge lists of classes? :kissing_closed_eyes: